### PR TITLE
chore(ci): add multiplier to get effective pbs throughput value

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: ci/benchmark_parser.py [hunk 1]
```diff
@@ -108,12 +108,17 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
                     )
                 )
 
+                # This is a special case where PBS are blasted as vector of 512 LWE ciphertext
+                # to saturate the machine. To get the actual throughput we need to multiply by the
+                # length of the vector.
+                multiplier = 512 if "PBS_throughput" in test_name else 1
+
```
Comment (IceTDrinker) [inline ci/benchmark_parser.py:115]: some benchmarks have different chunk sizes though ?

Comment (soonum) [reply ci/benchmark_parser.py:115]: This is the only one we've set up to saturate the machine.

Comment (IceTDrinker) [reply ci/benchmark_parser.py:115]: What I mean is that we have other throughput stuff, and this looks like a hack more than actual solution to compute throughput for all cases we are interested in

Comment (soonum) [reply ci/benchmark_parser.py:115]: Indeed, this is clearly a hack. That said, I haven't found a way at criterion level to pre-process the value.

Comment (IceTDrinker) [reply ci/benchmark_parser.py:115]: Can we detect « chunk » in the bench name ? I think it’s the only benchmarks with chunk in the name and then parse the chunk size from there

**Comment by soonum** [comment]
@slab-ci pbs_bench

File: ci/benchmark_parser.py [hunk 2]
```diff
@@ -108,12 +108,26 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
                     )
                 )
 
+                # This is a special case where PBS are blasted as vector LWE ciphertext with
+                # variable length to saturate the machine. To get the actual throughput we need to
+                # multiply by the length of the vector.
+                if "PBS_throughput" in test_name and ("chunk") in test_name:
```
Comment (IceTDrinker) [inline ci/benchmark_parser.py:114]: ("chunk") -> "chunk"

File: ci/benchmark_parser.py [hunk 3]
```diff
@@ -108,12 +108,26 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
...
+                    try:
+                        multiplier = int(test_name.split("chunk")[0].split("_")[-1])
+                        print("Name:", test_name, "Multi:", multiplier)
```
Comment (IceTDrinker) [inline ci/benchmark_parser.py:117]: remove the print which should not be needed

File: ci/benchmark_parser.py [hunk 4]
```diff
@@ -108,12 +108,26 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
...
+                    except ValueError:
+                        parsing_failures.append((full_name,
+                                                 f"failed to extract throughput multiplier"))
```
Comment (IceTDrinker) [inline ci/benchmark_parser.py:120]: f string without {}

**Review by IceTDrinker — APPROVED** [review]
Looks good thanks !

**Pull request merged**